include/common/font.h

```diff
@@ -4,11 +4,25 @@
 
 struct lab_data_buffer;
 
+enum font_slant {
+	FONT_SLANT_NORMAL,
+	FONT_SLANT_ITALIC
+};
+
+enum font_weight {
+	FONT_WEIGHT_NORMAL,
+	FONT_WEIGHT_BOLD
+};
+
 struct font {
 	char *name;
 	int size;
+	enum font_slant slant;
+	enum font_weight weight;
 };
 
+struct _PangoFontDescription *font_to_pango_desc(struct font *font);
```

Comment:
Is the use of struct here to avoid being pango in the includes?

Reply:
Yes, especially now that `font.h` is pulled in by `rcxml.h` as well

Reply:
Nice. It took me a few seconds before the penny dropped. Like it.